synapse/lib/stormtypes.py

```diff
@@ -4731,6 +4731,9 @@ def __eq__(self, othr):
             return str(self) == str(othr)
         return False
 
+    async def bool(self):
```

Comment:
Would it make sense for this to be the default behavior in a base class to avoid the need to have the same function all over?

Reply:
The other thing - changing the base Prim to use `bool(self.valu)` would be a behavior change for some objects who dynamically resolve their `value()` from storage data; notably in the case where a object was **deleted** but the runtime is still holding a reference to the object.